### PR TITLE
Add HTTP support to DataCentric commands

### PR DESCRIPTION
**Pull request by IonesioJunior**
## Description
Add support to the HTTP protocol in the data-centric command messages.

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [X] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [X] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [X] My changes are covered by tests

### CODE REVIEW

**Comment by jmaunon** [comment]
Hi!
Which is the goal of this PR? (I guess that is for sending syft command via http request) 
would it be possible/make sense to add a test?

File: apps/node/src/app/main/routes/data_centric/routes.py [hunk 1]
```diff
@@ -110,6 +114,29 @@ def list_workers():
 # ======= REST API =======
 
 
+@data_centric_routes.route("/syft/", methods=["POST"])
+@cross_origin()
+def syft_commands():
+    try:
```
Comment (jmaunon) [inline apps/node/src/app/main/routes/data_centric/routes.py:120]: missing docstring

**Review by jmaunon — COMMENTED** [review]
Hi!
Which is the goal of this PR? (I guess that is for sending syft command via http request)
would it be possible/make sense to add a test?

**Review by cereallarceny — APPROVED** [review]

File: apps/node/src/app/main/routes/data_centric/routes.py (hunk 1 above)
Comment (IonesioJunior) [reply apps/node/src/app/main/routes/data_centric/routes.py:125]: Done! :)

**Comment by IonesioJunior** [comment]
Creating unit tests right now!

**Pull request closed without merging**